controllers/configurationpolicy_controller.go

```diff
@@ -1627,8 +1627,16 @@ func handleAddingPolicy(plc *policyv1.ConfigurationPolicy) error {
 			}
 		}
 	}
-	selectedNamespaces := common.GetSelectedNamespaces(plc.Spec.NamespaceSelector.Include,
-		plc.Spec.NamespaceSelector.Exclude, allNamespaces)
+	//build namespace lists
+	exclude := []string{}
+	for _, ns := range plc.Spec.NamespaceSelector.Exclude {
+		exclude = append(exclude, string(ns))
+	}
+	include := []string{}
+	for _, ns := range plc.Spec.NamespaceSelector.Include {
+		include = append(include, string(ns))
+	}
+	selectedNamespaces := common.GetSelectedNamespaces(include, exclude, allNamespaces)
```

Comment:
Optional: You could modify the function signature of `common.GetSelectedNamespaces` so that it accepts slices of `NonEmptyString`. Since this is the only place it's called, I think that'd be a cleaner solution, but I won't block the PR on it.

Reply:
@mprahl if we pass the `NonEmptyString` into that function, we'll also have to modify `FindPattern` to accept `NonEmptyString`. Since that function uses `strings.Contains`, we would have to convert the `NonEmptyString` to a regular string in that function anyway so I think it makes more sense to just do the string conversion where it is in the PR